docs/user_documentation/import-data/guide-upload.md

```diff
@@ -1,24 +1,30 @@
-# Upload you data (advanced importer)
-In this section you will learn how to upload you data. Let us not wait any longer! After a quick re-cap of terminology and formats we will start off with the most basic step, importing your data.
+# Upload your data using the Import Wizard (Advanced data import)
+In this section you will learn how to upload your data. 
+Let us not wait any longer! 
```

Comment:
if we are updating the page i suggest to remove this line, gets old very quickly imo